content/webapp/components/ItemViewerContext/ItemViewerContext.tsx

```diff
@@ -30,6 +30,7 @@ type Props = {
   parentManifest: ParentManifest | undefined;
   searchResults: SearchResults | null;
   setSearchResults: (v) => void;
+  accessToken: string | undefined;
```

Comment:
np: below we use the `prop?: string` form, we could align? very much not a blocker or necessary.